Dockerfile

```diff
@@ -16,6 +16,11 @@ RUN zypper install -y \
     xorriso  \
     libguestfs kernel-default e2fsprogs parted gptfdisk btrfsprogs
 
+# TODO: Install nmc via zypper once an RPM package is available
```

Comment:
Can you add an issue to make sure we remember to readdress this? I'll start the discussion in chat about issue labels.

Reply:
We already track this internally in Jira. Do we want a Github issue for it as well?

Reply:
Raised an issue.